terraform/rob-cos/main.tf

```diff
@@ -2,37 +2,38 @@
 
 data "juju_model" "model" {
   name = var.model
+  owner = "admin"
 }
 
 # -------------- # Applications --------------
 
 module "blackbox_exporter" {
-  source   = "git::https://github.com/ubuntu-robotics/blackbox-exporter-k8s-operator//terraform?ref=feat/terraform"
+  source   = "git::https://github.com/ubuntu-robotics/blackbox-exporter-k8s-operator//terraform?ref=59bb50b8c41712499ad4071726160f0fed7cb232"
   app_name = "blackbox-exporter"
-  model    = data.juju_model.model.name
+  model_uuid    = data.juju_model.model.uuid
```

Comment:
Linting seems off here and at other places below.
Could you add some linting and validation test (see e.g. [here](https://github.com/canonical/observability-stack/blob/main/.github/workflows/terraform.yml)) either here or in another pr ?